test_dagmc_stats.py

```diff
@@ -127,3 +128,22 @@ def test_get_area_triangle():
     exp = my_core.get_entities_by_type(root_set, types.MBTRI).size()
     obs = len(dagmc_stats.get_area_triangle(my_core, root_set))
     assertEqual(exp,obs)
+    
+def test_get_roughness():
+    """
+    Tests part of the get_roughness function
+    """
+    test_input2 = "single-cube.h5m"
+    my_core = core.Core()
+    my_core.load_file(test_input2)
+    root_set = my_core.get_root_set()
+    entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
+    native_ranges = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)
+    
+    exp = 0
+    obs = get_roughness(my_core, native_ranges)[0]
```

Comment:
this function returns an entire list, correct? We should be testing if the entire list matches our expected result.